doc/api/events.md

```diff
@@ -216,6 +228,8 @@ but important side effect: any *additional* listeners registered to the same
 listener that is in the process of being added.
 
 ```js
+'use strict';
+
```

Comment:
This example both works without `use strict` and is partial (a continuation of the first example), that means that this is not the correct place for `use strict`.